src/demo/demo.stache

```diff
@@ -16,13 +24,16 @@
 	<can-import from="@tucows/donejs-carousel-plugin/demo/demo.less!" />
 	<can-import from="@tucows/donejs-carousel-plugin" />
 	<tucows-donejs-carousel
-		activeSlideIndex:to='./activeSlideIndex'
+		activeSlideIndex:bind='./activeSlideIndex'
```

Comment:
also just out of my own curiosity why the change here? what does this accomplish?

Reply:
In the TV channel component when we use the scroller to flip between channel types, I need to tell the carousel to go back to slide 0. This lets me do that.